docs/src/components/App.tsx

```diff
@@ -37,8 +39,17 @@ export class App extends React.Component<{}, {}> {
     }
 
     private renderRemoteStream(stream: ConferenceStream) {
+        // NOTE(yunsi): Use the fisrt 10 characters as the remote name
+        // TOEO(yunsi): Find a better way to define remote name
+        const name = stream.id.substring(0, 10);
```

Comment:
What kind of name need to display?

Reply:
In spreed every remote video has their user name displayed at the bottom of the video, the user name will turn green if that user is speaking.